src/components/plots/CategoricalEmbeddingPlot.jsx

```diff
@@ -89,7 +95,7 @@ const CategoricalEmbeddingPlot = (props) => {
 
     return (
       <center>
-        <Vega spec={plotSpec} renderer='canvas' actions={actions} />
+        <Vega spec={plotSpec} renderer='canvas' actions={actions} scaleFactor={2} />
```

Comment:
what is this `scaleFactor`?

Reply:
Anastasija messaged me yesterday asking if we can improve the resolution of the plots. This property is used to set the scale of the image downloaded from the plot. I'm including this so that it can be staged and she can try it out in staging. If it's good, we can implement it for all plots.

Reply:
Anastasija have confirmed that using this property makes the plot resolution better. I don't want to add this property together with this ticket as it will cause a lot of small changes to a lot of files. I've created a new ticket to do this, with a general clean up for the plot actions: https://biomage.atlassian.net/browse/BIOMAGE-1782